server/src/tree-sitter/analyzer.ts

```diff
@@ -587,6 +580,34 @@ export default class Analyzer {
     return fileUris
   }
 
+  public findFilesInProjectScanner (filePath: string): ElementInfo[] {
```

Comment:
This will probably not work with docker containers which can change the mount points name for sources and build directories. The proper way to open URIs reported by bitbake is shown in https://github.com/yoctoproject/vscode-bitbake/blob/fbb0d49c984ec1a4011be0c6d14aa3a541a5c585/client/src/ui/BitbakeCommands.ts#L381

Reply:
You should test your code with [kas-container](https://github.com/siemens/kas/blob/master/kas-container) as bitbake.commandWrapper.